services/consensusalgo/benchmarkconsensus/leader.go

```diff
@@ -182,7 +182,7 @@ func (s *service) leaderHandleCommittedVote(sender *gossipmessages.SenderSignatu
 
 	// count if we have enough votes to move forward
 	existingVotes := len(s.lastCommittedBlockVoters) + 1
-	s.reporting.Info("Vote arrived", instrumentation.Int("existing-votes", existingVotes), instrumentation.Int("required-votes", s.requiredQuorumSize()))
+	s.reporting.Info("Valid vote arrived", instrumentation.Int("existing-votes", existingVotes), instrumentation.Int("required-votes", s.requiredQuorumSize()))
```

Comment:
Maybe it would be beneficial to log who voted and how.

Reply:
We have a log on the sender side when the vote is sent, in the recipient side we have a log on the transport arrived (which shows the message) and then this general log that the vote was approved. I used them all to debug this and it works pretty well